django/thunderstore/api/cyberstorm/views/package_deprecate.py

```diff
@@ -1,12 +1,10 @@
 from drf_yasg.utils import swagger_auto_schema
 from rest_framework import serializers, status
-from rest_framework.exceptions import PermissionDenied
-from rest_framework.generics import get_object_or_404
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
-from thunderstore.repository.models import Namespace, Package
+from thunderstore.api.cyberstorm.services import package as package_services
```

Comment:
Not the biggest fan of a convention that requires renaming imports outside of name clashes (this somewhat obfuscates what code references are pointing to due to the rename & makes auto-import unintuitive with IDEs, etc)

So while I don't find it an issue for single-off use cases, if this is something you foresee having to do a lot in the future with the service pattern, I'd recommend coming up with a way to avoid it (i.e. just renaming the module itself instead of what it's imported as at import site)